fluent-bit/deb/debian/copyright

```diff
@@ -0,0 +1,21 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: fluent-bit
+Source: https://github.com/fluent/fluent-bit
+
+Files: *
+Copyright: 2018-2020 Michael Ray <michael.ray@arm.com>
```

Comment:
2021 here?  also, should be copyright holder be Pelion?

Reply:
will change